scripts/check_duplicates.py

```diff
@@ -64,9 +64,11 @@ def create_default_sources():
     return sources
 
 
-def check_duplicates(sources):
+def check_duplicates(sources, tag):
```

Comment:
Please pass `ros_distro_name`, `os_name`, and `os_codename` as separate arguments. Otherwise the argument `tag` is unclear what it represents.